sdk/agrifood/agrifood-farming-rest/package.json

```diff
@@ -86,7 +86,7 @@
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
     "@azure-rest/core-client": "1.0.0-beta.4",
-    "@azure/core-rest-pipeline": "^1.0.3",
+    "@azure/core-rest-pipeline": "^1.1.0",
```

Comment:
This is a REST level client and so this file is auto generated. Therefore, we should update the code gen as well. Can we get an issue logged at https://github.com/Azure/autorest.typescript/issues

Reply:
Or you could pile on to https://github.com/Azure/autorest.typescript/issues/1048

cc @sarangan12